airflow-core/src/airflow/ui/public/i18n/locales/es/components.json

```diff
@@ -0,0 +1,131 @@
+{
+  "backfill": {
+    "affected_one": "1 ejecución será activada.",
+    "affected_other": "{{count}} ejecuciones serán activadas.",
+    "affectedNone": "No hay ejecuciones que coincidan con los criterios seleccionados.",
+    "backwards": "Ejecutar Hacia Atrás",
+    "dateRange": "Rango de Fechas",
+    "dateRangeFrom": "Desde",
+    "dateRangeTo": "Hasta",
+    "errorStartDateBeforeEndDate": "La Fecha de Inicio debe ser antes de la Fecha de Fin",
+    "maxRuns": "Máximo de Ejecuciones Activas",
+    "reprocessBehavior": "Comportamiento de Reprocesamiento",
+    "run": "Ejecutar Backfill",
+    "selectDescription": "Ejecutar este DAG para un rango de fechas",
+    "selectLabel": "Backfill",
+    "title": "Ejecutar Backfill",
+    "toaster": {
+      "success": {
+        "description": "Backfill jobs han sido activados exitosamente.",
+        "title": "Backfill generado"
+      }
+    },
+    "tooltip": "Backfill requiere una programación",
+    "unpause": "Reanudar {{dag_display_name}} al activarse",
+    "validation": {
+      "datesRequired": "Ambos Data Interval Start Date y End Date deben ser proporcionados.",
```

Comment:
here we use the english term for start and end date?

Reply:
I believe the "data" in data interval refers to the data being processed and not that it's about dates. 🤔  i might be wrong though. @bbovenzi what do you think?

Reply:
Yes, that's the time component is included in "interval". The term "data" is unrelated to the term "date" or time here. It's similar to how Data Engineer translates to "Ingeniero/a de datos".